common/recommender/Baseline.js

```diff
@@ -30,11 +32,15 @@ class Baseline {
        imageCount, isBookmarked, hasDescription],
       // Features weights: Positive values decrease the score proportional to a factor of feature * weight.
       //                   Negative values increase score proportional to a factor of feature * weight.
-      [0.4, 0.7, 0.1, -0.4, -0.2, -0.1]);
+      this.options.highlightsCoefficients || COEFFICIENTS);
 
     return Object.assign({}, entry, {score}, {host});
   }
 
+  updateOptions(options = {}) {
+    this.options = options;
```

Comment:
i'm assuming the only time you're ever going to call ```updateOptions``` is when you need to pass in new options - so i think it's safe to assume you'll have a param here, so you can remove ```= {}```